docs/source/integrations/burrow_example.rst

```diff
@@ -150,85 +150,120 @@ Note the last line of the send function that emits a Sent event which gets recor
+This contract calls the *getAssetBalance*, *createAccount*, *addAsset* and *transferAsset* methods of the Iroha *ServiceContract* API.
 
-Case 3. Changing Iroha state
-----------------------------
-
-The final example we consider here is a transfer of an asset from one Iroha account to another.
+We need to compile the contract above to get the bytecode using a full-fledged Solidity compiler or the Web-based *Remix IDE*.
+Now, we can send transactions from the Python Iroha client to deploy the contract to the EVM and also to call the different functions of the contact.
+The contract is deployed in a similar manner as shown above. To call a function of the deployed contract, function signature and it's arguments must be encoded following the ABI specifications.
 
+Here is a sample python code that calls a function of a deployed contract:
 
-The contract code is as follows:
-
-.. code-block:: solidity
-
-	contract Transfer {
-	    address public serviceContractAddress;
-
-	    event Transferred(string indexed source, string indexed destination, string amount);
-
-	    // Initializing service contract address in constructor
-	    constructor() public {
-	        serviceContractAddress = 0xA6Abc17819738299B3B2c1CE46d55c74f04E290C;
-	    }
-
-	    // Queries the balance in _asset of an Iroha _account
-	    function transferAsset(string memory src, string memory dst,
-	                           string memory asset, string memory amount) public
-	                    returns (bytes memory result) {
-	        bytes memory payload = abi.encodeWithSignature(
-	            "transferAsset(string,string,string,string)",
-	            src,
-	            dst,
-	            asset,
-	            amount);
-	        (bool success, bytes memory ret) =
-	            address(serviceContractAddress).delegatecall(payload);
-	        require(success, "Error calling service contract function");
-
-	        emit Transferred(src, dst, amount);
-	        result = ret;
-	    }
-	}
+.. code-block:: python
 
+	def add_asset(address):
+		params = integration_helpers.get_first_four_bytes_of_keccak(
+			b"addAsset(string,string)"
+		)
+		no_of_param = 2
+		for x in range(no_of_param):
+			params = params + integration_helpers.left_padded_address_of_param(
+				x, no_of_param
+			)
+		params = params + integration_helpers.argument_encoding("coin#test")  # asset id
+		params = params + integration_helpers.argument_encoding("500")  # amount of asset
+		tx = iroha.transaction(
+			[
+				iroha.command(
+					"CallEngine", caller=ADMIN_ACCOUNT_ID, callee=address, input=params
+				)
+			]
+		)
+		IrohaCrypto.sign_transaction(tx, ADMIN_PRIVATE_KEY)
+		response = net.send_tx(tx)
+		for status in net.tx_status_stream(tx):
+			print(status)
+
+For more examples and how the code works, you can visit `here  <https://github.com/hyperledger/iroha/tree/support/1.2.x/example/burrow_integration>`_ .
```

Comment:
I know that the PR is not integrated yet, but I suggest to change the link into:
https://github.com/hyperledger/iroha/tree/main/example/burrow_integration
```suggestion
For more examples and how the code works, you can visit `here  <https://github.com/hyperledger/iroha/tree/main/example/burrow_integration>`_ .
```
because soon the branch `1.2.x` won't be updated, so if the code would be updated link would move into old version.